_eo_021_enemies/shelled_dragon.md

```diff
@@ -5,11 +5,22 @@ image: orthoshelled_dragon.png
 start_floor: 28
 end_floor: 29
 agro: Sight
+hp: 138400
+attack_damage: 2976
 attack_type: Physical
 vulnerabilities:
   bind: false
   heavy: false
   sleep: true
   slow: true
   stun: false
+abilities:
+  - name: Diamondback
+    potency: n/a
+    description: 'grants permanent vulnerability down (33%) to self; can be
+    interrupted. Used immediately after pull'
```

Comment:
Not always! I thought this too until I wasted an interrupt on Tortoise Stomp

Reply:
Yikes! Fixed.